src/ServiceFabric/ServiceFabric/ServiceFabric.csproj

```diff
@@ -16,4 +16,11 @@
     <PackageReference Include="Microsoft.Azure.Management.ServiceFabric" Version="1.1.1" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="Template\Linux\parameter.json" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
Are these files required as part of the ServiceFabric module? If so, can we add a test that would prevent regressing on this in the future?

Reply:
yes, I'll add the test thanks.

Reply:
test added